### DISCUSSION TITLE
Some songs play, and some do not.

### DISCUSSION THREAD

**Discussion by Lynwood** [vicwomg/pikaraoke#353] [General]
I've added multiple files to the queue, and some files get "skipped over".  It lists them on the screen as coming next, and then skips it and moves on to the next song.  Other files of the same Karaoke type will play just fine, but others will not.  The same file that wont play, or was skipped will play just fine in VLC media player.  I can provide files for testing if needed.

**Comment by rosbusterus** [comment]
I had the same issue.
Songs downloaded and could be played in VLC, but skipped playing in Pikaraoke.
Today I've updated to latest YouTube dll (2024.08.06) now it's working again fine (previous I had YouTube dll of end of July installed).
